src/components/WhaleInfo/AmazingWhaleFacts.ts

```diff
@@ -0,0 +1,67 @@
+export function amazingWhaleFacts() {
```

Comment:
Can we add an interface for the facts and add the return type to this function?
```suggestion
interface Fact {
    header: string;
    detail: string;
}

export function amazingWhaleFacts(): Array<Fact>  {
```